subscription/controls.py

```diff
@@ -296,15 +296,30 @@ def chargeForCyVerse(stripe_api_key, stripe_token, priceToCharge, stripeDescript
         partnerObj = termObj.partnerId
         partnerId = partnerObj.partnerId
         status = True
+        #display error boolean for error capturing 
+        displayError = False
 
```

Comment:
We can add one additional variable, errMsg, for all the ones that have message['message'], it will also have errMsg, but for duplicate user cases, it only has errMsg but no message['message']. Then log based on errMsg